lib/ruby_lsp/requests/code_lens.rb

```diff
@@ -34,17 +34,18 @@ def provider
 
       sig do
         params(
+          global_state: GlobalState,
           uri: URI::Generic,
           dispatcher: Prism::Dispatcher,
         ).void
       end
-      def initialize(uri, dispatcher)
+      def initialize(global_state, uri, dispatcher)
         @response_builder = T.let(
           ResponseBuilders::CollectionResponseBuilder[Interface::CodeLens].new,
           ResponseBuilders::CollectionResponseBuilder[Interface::CodeLens],
         )
         super()
-        Listeners::CodeLens.new(@response_builder, uri, dispatcher)
+        Listeners::CodeLens.new(@response_builder, global_state, uri, dispatcher)
```

Comment:
We need to pass the global state to code lens listeners so that they can check the test framework.